tests/subtests/subtest.py

```diff
@@ -15,86 +15,99 @@ def plot(line, **kwargs):
 
 # hash match
 
-@pytest.mark.mpl_image_compare()
+@pytest.mark.image
+@pytest.mark.hash
+@pytest.mark.mpl_image_compare(savefig_kwargs={'metadata': {'Software': None}})
```

Comment:
@ConorMacBride @Cadair  Should I also adopt this pattern as a default in #160 ?

Seems like a sensible thing to do, right?

Reply:
Yeah, I think that would be a good idea. Then we can remove the `metadata` setting from this file.